content/applications/services/project/project_management.rst

```diff
@@ -0,0 +1,95 @@
+==================
+Project management
+==================
+
+Odoo Project uses the **Kanban** project management system. This means all projects are broken down
+into tasks, which are categorized on a whiteboard according to what production phase they are in.
+
+.. admonition:: Did you know?
+
+   The word **Kanban** comes from Japanese and refers to the "visual board" management method.
+
+.. seealso::
+   `Odoo Tutorials: Kanban Project Management
+   <https://www.odoo.com/slides/slide/kanban-project-management-1664>`_
+
+Configuration
+=============
+
+Open the **Project** app and click :guilabel:`Create` to start a new project. Enter a
+:guilabel:`Name` for your project and click :guilabel:`Create Project`.
+
+You can customize your existing **projects** from the dashboard by clicking the drop-down toggle
+button (:guilabel:`⋮`) on your project's **card**.
+
+.. image:: project_management/project-settings.png
+   :align: center
+   :alt: Project card
+
+This enables a new menu divided into four parts:
+
+- **View**: see an overview of your project's components, such as its tasks, milestones and project
+  updates. Depending on which apps you have activated, more options may be available, such as
+  **Documents**: all uploaded files can be found under this menu, as well as in the **Documents**
```

Comment:
I'm not loving this : inside a : 
![image](https://user-images.githubusercontent.com/95279628/214272694-26345fa8-0b3f-4299-ace8-8a58a5a31a25.png)
but that's nitpicking